### ISSUE TITLE
Does not work when HTTPS is enabled on NodeRed

### ISSUE THREAD

**Issue by Flight777** [Supergiovane/node-red-contrib-sonospollytts#24]
Hi!

Thanks so much for this amazing idea and add-on, if this work you've made my home automation complete :)

I was testing it today and everything seems to be set-up correctly, however, nothing played from the Sonos speaker. The MP3 files are generated in the \tmp folder, so that works fine.

I checked the node-red logs and noticed:

```
10 Nov 10:42:55 - [info] SonosPollyTTS: DEBUG - fromcache : /data/tmp/Hello_pause_ this is a test_em__Amy.mp3

10 Nov 10:42:55 - [info] SonosPollyTTS: PlaySonos - _songuri: /data/tmp/Hello_pause_ this is a test_em__Amy.mp3 sUrl: http://192.168.31.10:1880/tts/tts.mp3?f=%2Fdata%2Ftmp%2FHello_pause_%20this%20is%20a%20test_em__Amy.mp3
```

My Node-Red is running in **https**, thus the sURL will never work. I cannot find where I'm able to adjust this in the configuration of the sonospollytts node. Is this something that can be adjusted/build in? Would this also be the cause why I'm not hearing anything from my Sonos speaker? :)

Thanks!

**Comment by Supergiovane** [comment]
Hello Tripleseven, 
thank you.
There is a currently issue running in HTTPS.
To confirm that, please revert to http and try again.
With https involved, there's a problem in developing the SSL certificates that must be issued to Sonos, to be able to connect to the node.
Sorry for that!

**Comment by Flight777** [comment]
Yep, it works with HTTP :)

Shame it doesn't work with https because my Node-Red is accessible from the outside world for RedBot etc.

It is not possible to enable https and force the ignore of any certificate errors? Or is this on Sonos side?

**Comment by Supergiovane** [comment]
That’s a Sonos side. I’m working on it, but for now i’ts a no-go

**Comment by Supergiovane** [comment]
Fixed in 1.1.0
